### PR TITLE
Remove no longer published content from the idx

### PR DESCRIPTION
**Pull request by mbst-jamie**

### CODE REVIEW

File: atlas-elasticsearch/src/main/java/org/atlasapi/content/EsContentIndex.java [hunk 1]
```diff
@@ -430,6 +439,38 @@ protected Void visitContainer(Container container) {
         }
     }
 
+    private void unindexNoLongerPublishedContent(Content content) {
```
Comment (tfm) [inline atlas-elasticsearch/src/main/java/org/atlasapi/content/EsContentIndex.java]: perhaps just `removeFromIndex(Content content)`

**Comment by tfm** [comment]
Test?

**Comment by tfm** [comment]
Back to you, @mbst-jamie

File: atlas-elasticsearch/src/main/java/org/atlasapi/content/EsContentIndex.java (hunk 1 above)
Comment (mbst-jamie) [inline atlas-elasticsearch/src/main/java/org/atlasapi/content/EsContentIndex.java]: This can also be de-duped a bit... ty

**Pull request merged**